src/utils/media-utils.js

```diff
@@ -119,7 +119,7 @@ export const addMedia = (src, template, contentOrigin, resolve = false, resize =
     entity.addEventListener(eventName, () => {
       clearTimeout(fireLoadingTimeout);
 
-      if (!entity.classList.contains("pen")) {
+      if (!entity.classList.contains("pen") && !entity.getAttribute("animation__spawn-start")) {
```

Comment:
Ah interesting, I expected that the animation component would remove itself. I guess it doesn't so this makes sense. Wonder if pagination should have its own loading animation at some point